tests.py

```diff
@@ -33,7 +33,8 @@
 from libarchive import Archive, is_archive_name, is_archive
 from libarchive.zip import is_zipfile, ZipFile, ZipEntry
 
-PY3 = sys.version_info[0] == 3
+# PY3 = sys.version_info[0] == 3
+PY3 = True
```

Comment:
We run the tests in python 2 and 3. Why shouldn't this detect that?

Reply:
Done